release-tools/mktarball.sh

```diff
@@ -6,11 +6,17 @@ if [ -z $VERSION ]; then
 	exit 1
 fi
 DESTDIR=tarsnap-autoconf-${VERSION}
+RELEASEDATE=`date "+%B %d, %Y"`
 
 # Copy bits in
 mkdir ${DESTDIR}
 cp COPYING Makefile.am actarsnap.m4 configure.ac tsserver .autom4te.cfg ${DESTDIR}/
 cp -R keygen keymgmt keyregen lib libarchive libcperciva misc pkg recrypt tar ${DESTDIR}/
+# Copy with substitution
+for MANPAGE in */*-man */*-mdoc */*-man.in */*-mdoc.in
+do
```

Comment:
Huh.  I suppose this is perfectly valid `sh` syntax, but I've *always* seen it written as `for foo in bar baz; do` on a single line.  Might as well be consistent with 99%+ of other shell scripts.